src/datastore_export/load_datastore_backups_to_big_query_service.py

```diff
@@ -80,23 +86,24 @@ def __is_finished_with_success(self, load_job_id):
         logging.info("Export from GCS to BQ finished successfully.")
         return True
 
-    def __wait_till_done(self, load_job_id):
+    def __wait_till_done(self, load_job):
         while True:
             result = self.big_query.get_job(
                 project_id=configuration.backup_project_id,
-                job_id=load_job_id
+                job_id=load_job.job_id,
+                location=load_job.location
             )
             if 'errors' in result['status']:
                 raise LoadDatastoreBackupsToBigQueryException(
-                    "Export from GCS to BQ failed, job id: {}".format(
-                        load_job_id)
+                    "Export from GCS to BQ failed, job id: {}, location: {}"
```

Comment:
Please implement `__str__` method in LoadJob and use it here

Reply:
done, but logic is moved into BigQueryJobReference class